ckanfunctionaltests/api/test_paging.py

```diff
@@ -95,7 +95,17 @@ def test_search_paging_equivalence(
     count_getter,
     limit_param,
     offset_param,
+    variables
 ):
+    if variables.get('ckan_version') == '2.9':
```

Comment:
Might be worth linking in the commit ae1acfb5281bd845556869e3fa2c4c9f229d531f some evidence to show that "v1 API endpoint has been dropped in 2.9". Saves a developer having to dig around the web and try to find it.

Reply:
I'll link to the CKAN changelog